<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NutritionDialog</class>
 <widget class="QDialog" name="NutritionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор КБЖУ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 16px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Калькулятор КБЖУ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="inputGroupBox">
     <property name="title">
      <string>Параметры</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="weightLabel">
        <property name="text">
         <string>Вес (кг):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="weightSpinBox">
        <property name="minimum">
         <double>30.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="value">
         <double>70.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="goalLabel">
        <property name="text">
         <string>Цель:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="goalComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="activityLabel">
        <property name="text">
         <string>Активность:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="activityComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calculateButton">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="resultGroupBox">
     <property name="title">
      <string>Результаты</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="proteinResultLabel">
        <property name="text">
         <string>Белки:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="proteinLabel">
        <property name="text">
         <string>0 г</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fatsResultLabel">
        <property name="text">
         <string>Жиры:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="fatsLabel">
        <property name="text">
         <string>0 г</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="carbsResultLabel">
        <property name="text">
         <string>Углеводы:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="carbsLabel">
        <property name="text">
         <string>0 г</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="caloriesResultLabel">
        <property name="text">
         <string>Калории:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="caloriesLabel">
        <property name="text">
         <string>0 ккал</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calculateButton</sender>
   <signal>clicked()</signal>
   <receiver>NutritionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
